.ci/special-jobs/php71.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -xe
+# change php-fpm version in docker images of php-fpm and all microservice containers
+sed -i "s/php\:7\.2\-fpm\-alpine*/php\:7\.1\-fpm\-alpine/" $WORKSPACE/project-base/docker/php-fpm/Dockerfile
+sed -i "s/php\:7\.2\-fpm\-alpine*/php\:7\.1\-fpm\-alpine/" $WORKSPACE/microservices/*/docker/Dockerfile
```

Comment:
Maybe, i would extract these types of configuration modifications into something like `change-php-version.sh` and then execute kubernetes-buildpack after it in Jenkins